localstack/config.py

```diff
@@ -406,6 +406,27 @@ def in_docker():
 # TODO: this will likely become the default and may get removed in the future
 FORWARD_EDGE_INMEM = True
 
+
```

Comment:
This section is moved further up the file so we have access to the logger by the time we parse `GATEWAY_LISTEN`